lib/pure/asynchttpserver.nim

```diff
@@ -41,10 +41,14 @@ runnableExamples("-r:off"):
 
 import asyncnet, asyncdispatch, parseutils, uri, strutils
 import httpcore
+from nativesockets import getLocalAddr, AF_INET
 import std/private/since
 
 export httpcore except parseHeader
 
+since (1, 5, 1):
+  export Port
```

Comment:
No. People should know how to get the `Port` type.

Reply:
done; (note that asyncdispatch also re-exports Port, but that can be discussed later)